sunpy/sun/constants.py

```diff
@@ -134,5 +139,8 @@ def _build_docstring():
 # Observable parameters
 average_angular_size = get('average angular size')
 sidereal_rotation_rate = get('sidereal rotation rate')
+
 first_carrington_rotation = Time(get('first Carrington rotation (JD TT)'), format='jd', scale='tt')
+first_carrington_rotation.__doc__ = 'Time of the start of the first Carrington rotation'
```

Comment:
Does not doing this work?
```suggestion
"""Time of the start of the first Carrington rotation"""
```

Reply:
I used the same syntax as above (`#:` above the instance).